pipeline/clinvar/enigma_from_clinvar.py

```diff
@@ -0,0 +1,291 @@
+import multiprocessing
+import re
+from multiprocessing.pool import ThreadPool
+
+import click
+import pandas as pd
+from lxml import etree
+
+import clinvar
+import hgvs_utils
+
+default_val = None
+
+MULTI_ENTRY_SEP = ','
+
+
+def _get_clinvar_sets(fin):
+    f = etree.parse(fin)
+    root = f.getroot()
+    return root.xpath('//ClinVarSet')
+
+
+def _is_bic_designation(s):
+    return any(k in s.lower() for k in {'ins', 'del', 'dup'}) or \
+           ('>' in s and not s.startswith('p.')) or \
```

Comment:
if it starts with 'p.' won't it be true anyways?
```suggestion
           ('>' in s or \
```

Reply:
yes, I guess. anyways the filter is incorrect, I've fixed in in a subsequent version